src/app.module.ts

```diff
@@ -131,18 +135,22 @@ export class AppModule implements NestModule {
         }),
         TypeOrmModule.forRootAsync({
           imports: [ConfigModule],
-          useFactory: async (configService: ConfigService) => {
-            const typeormConfig = await configService.getOrThrow('typeorm');
+          useFactory: async (
+            configService: ConfigService,
+            loggingService: ILoggingService,
+          ) => {
+            const typeormConfig = configService.getOrThrow('db.orm');
             const postgresConfigObject = postgresConfig(
-              await configService.getOrThrow('db.postgres'),
+              await configService.getOrThrow('db.connection.postgres'),
```

Comment:
```suggestion
             configService.getOrThrow('db.connection.postgres'),
```

Reply:
Done https://github.com/safe-global/safe-client-gateway/commit/2ea0e5d8a752fd31cff745ff3490d8fe8134ddc7

Reply:
It [doesn't seem to have changed](https://github.com/safe-global/safe-client-gateway/blob/ff1c5a4aad788e1f0929ba9f5653313d85e8d42c/src/app.module.ts#L144).

Reply:
Sorry for the confusion. Somehow the change was reverted on my end. Done
https://github.com/safe-global/safe-client-gateway/pull/1988/commits/e907e3cc5db5e7b605fce8afb51ea6464a464252